op-node/rollup/driver/step.go

```diff
@@ -389,10 +390,12 @@ func (d *outputImpl) insertHeadBlock(ctx context.Context, fc eth.ForkchoiceState
 		d.log.Error("Dropped deposits when executing L2 block")
 	}
 
-	err = d.l2.NewPayload(ctx, payload)
-	if err != nil {
+	if status, err := d.l2.NewPayload(ctx, payload); err != nil {
+		return nil, fmt.Errorf("failed to insert execution payload: %w", err)
+	} else if err := eth.ForkchoiceUpdateErr(*status); err != nil {
```

Comment:
```suggestion
	} else if err := eth.NewPayloadErr(payload, status); err != nil {
```

Reply:
updated as well in the other place that does this